common/Reducers.jsm

```diff
@@ -484,6 +484,37 @@ function DiscoveryStream(prevState = INITIAL_STATE.DiscoveryStream, action) {
           spocs_endpoint: action.data || INITIAL_STATE.DiscoveryStream.spocs.spocs_endpoint,
         },
       };
+    case at.PLACES_LINK_BLOCKED:
+      // Return if action data is empty, or spocs or feeds data is not loaded
+      if (!action.data || !prevState.spocs.loaded || !prevState.feeds.loaded) {
+        return prevState;
+      }
+      // Return if spocs is undefined in prevState.
+      if (!(prevState.spocs.data && prevState.spocs.data.spocs)) {
```

Comment:
This check impacts if feeds are present, updating PR to check for empty spocs data inside return